main/EventSystem.h

```diff
@@ -18,6 +18,8 @@ extern "C" {
 #include "LuaCommon.h"
 #include "concurrent_queue.h"
 
+#define DZVENTS_VERSION "2.3.0"
```

Comment:
Please do not make changes to a header file if you need a version number.
Best to do this in the CPP.
Better to have a dzversion.h, and include this in the class it's cpp file (if there is a dz class, in this class, else in EventSystem.cpp)

Reply:
(otherwise at a version change we need to recompile a lot more then one class)

Reply:
Where should we put this file? Like appversion.h in the root?

Reply:
It's being used by webserver, so including it in eventsystem.cpp doesn't work (there is no dz class). Putting it in eventsystem.h works, is that ok? For now I've put it in the root next to appversion.h, let me know if you want it elsewhere (in main?).

Reply:
Maybe write a wrapper class for dzVents and in this class make a funciton like

char *GetVersion()

that returns the version that is declared in the .cpp file

Maybe also move some stuff from the base EventSystem to this class ?